requirements/functests.in

```diff
@@ -4,5 +4,6 @@ pip-sync-faster
 httpretty
 pytest
 factory-boy
+pytest-factoryboy
```

Comment:
LMS has its own custom integration of `factory_boy` into `pytest`. It won't do any harm to also have `pytest-factoryboy` here: LMS doesn't have to use it and can continue to use its own custom thing or can use both at once. At some point I think we should convert all existing factories to `pytest-factoryboy` and remove the custom integration